src/game.css

```diff
@@ -1,11 +1,23 @@
+.game-grids {
+    display:flex;
+    justify-content:center
+}
+
 .game-board {
+    height: 600px; 
+}
+.block-display {
+    height: 300px; 
+    position: relative;
+    
+   
+}
+.grid {
     display: block;
-    border: 1px solid #000000; 
+    border: 4px solid #000000; 
```

Comment:
Was this intentional?

Reply:
Yes, Dimi and I thought this might look better